resources/views/secretariat/document/list.blade.php

```diff
@@ -0,0 +1,49 @@
+@extends('layouts.app')
+
+@section('title')
+<a href="#!" class="breadcrumb">@lang('admin.admin')</a>
+<a href="#!" class="breadcrumb">@lang('admin.document_requests')</a>
+@endsection
+@section('secretariat_module') active @endsection
+
+@section('content')
+<div class="row">
+    <div class="col s12">
+        <div class="card">
+            <div class="card-content">
+                <span class="card-title">@lang('admin.document_requests')</span>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>@lang('user.name')</th>
+                            <th>@lang('document.request')</th>
+                            <th>@lang('checkout.date')</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        @foreach ($requests as $request)
+                            @if ($request->user)
+                            <tr>
+                                <td>
+                                    {{ $request->user->name ?? 'Name not found' }}
+                                </td>
+                                <td>
+                                    @if ($request->document_type === 'StatusCertificate')
+                                        @lang('document.status-cert')
+                                    @else
+                                        {{ $request->document_type }}
+                                    @endif
+                                </td>
+                                <td>
+                                    {{ $request->created_at->format('Y-m-d h:m') }}
+                                </td>
+                            </tr>
```

Comment:
Please add a 'done' button and controller function that deletes the request.
Optional: send back an email to the user here (you need to create a new email template). - or create a new issue for this.